### PR TITLE
enh(mc): automatic commit for network-routers-juniper-mseries-snmp

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 1]
```diff
@@ -2,48 +2,820 @@
...
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Juniper-Mseries-SNMP-custom" label="Net-Juniper-Mseries-SNMP-custom">
+
+| Alias          | Modèle de service                         | Description                                        |
+|:---------------|:------------------------------------------|:---------------------------------------------------|
+| Cpu-Routing    | Net-Juniper-Mseries-Cpu-Routing-custom    | Contrôle l'utilisation CPU du 'routing engine'     |
+| Hardware       | Net-Juniper-Mseries-Hardware-custom       | Contrôle l'état du matériel                        |
+| Memory-Routing | Net-Juniper-Mseries-Memory-Routing-custom | Contrôle l'utilisation mémoire du 'Routing Engine' |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:26]:
```suggestion
| Memory-Routing | Net-Juniper-Mseries-Memory-Routing-custom | Contrôle l'utilisation mémoire du 'routing engine' |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 2]
```diff
@@ -2,48 +2,820 @@
...
+
+> Les services listés ci-dessus sont créés automatiquement lorsque le modèle d'hôte **Net-Juniper-Mseries-SNMP-custom** est utilisé.
+
+</TabItem>
+<TabItem value="Non rattachés à un modèle d'hôte" label="Non rattachés à un modèle d'hôte">
+
+| Alias                      | Modèle de service                                          | Description                                                                                                      | Découverte |
+|:---------------------------|:-----------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------|:----------:|
+| Bgp-Peer-Prefix-Statistics | Net-Juniper-Mseries-SNMP-Bgp-Peer-Prefix-Statistics-custom | Contrôle les statistiques des prefix des pairs BGP                                                               |            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:35]:
```suggestion
| Bgp-Peer-Prefix-Statistics | Net-Juniper-Mseries-SNMP-Bgp-Peer-Prefix-Statistics-custom | Contrôle les statistiques des préfixes des pairs BGP                                                               |            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 3]
```diff
@@ -2,48 +2,820 @@
...
+|:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Juniper-Mseries-SNMP-custom** host template |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                  | Description                                                             |
+|:---------------------------------|:------------------------------------------------------------------------|
+| Net-Juniper-Mseries-Storage-Name | Discover the disk partitions and monitor space occupation               |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:68]: English + French?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 4]
```diff
@@ -2,48 +2,820 @@
...
+| Net-Juniper-Mseries-Traffic-Name | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:69]:
```suggestion
| Net-Juniper-Mseries-Traffic-Name | Découvre les interfaces réseau et supervise le statut et l'utilisation |
```

File: pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 5]
```diff
@@ -2,48 +2,819 @@
...
+* **Net-Juniper-Mseries-SNMP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Juniper-Mseries-SNMP-custom" label="Net-Juniper-Mseries-SNMP-custom">
+
+| Service Alias  | Service Template                          | Service Description                  |
+|:---------------|:------------------------------------------|:-------------------------------------|
+| Cpu-Routing    | Net-Juniper-Mseries-Cpu-Routing-custom    | Check CPU Usage of routing engine    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:23]:
```suggestion
| Cpu-Routing    | Net-Juniper-Mseries-Cpu-Routing-custom    | Check CPU usage of routing engine    |
```

File: pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 6]
```diff
@@ -2,48 +2,819 @@
...
+| Hardware       | Net-Juniper-Mseries-Hardware-custom       | Check hardware                       |
+| Memory-Routing | Net-Juniper-Mseries-Memory-Routing-custom | Check Memory Usage of routing engine |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:25]:
```suggestion
| Memory-Routing | Net-Juniper-Mseries-Memory-Routing-custom | Check memory usage of routing engine |
```

File: pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 7]
```diff
@@ -2,48 +2,819 @@
...
+| Lsp-Status                 | Net-Juniper-Mseries-SNMP-Lsp-Status-custom                 | Check LSP status                                                                             |            |
+| Rsvp-Session-Status        | Net-Juniper-Mseries-SNMP-Rsvp-Session-Status-custom        | Check RSVP sessions status                                                                   |            |
+| Traffic-Generic-Id         | Net-Juniper-Mseries-Traffic-Generic-Id-custom              | Check the bandwidth of the interface. For each checks the name of the interface will appear  |            |
+| Traffic-Generic-Name       | Net-Juniper-Mseries-Traffic-Generic-Name-custom            | Check the bandwidth of the interface. For each checks the name of the interface will appear  |            |
+| Traffic-Global             | Net-Juniper-Mseries-Traffic-Global-custom                  | Check the bandwidth of  interfaces. For each checks the name of the interface will appear    | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:44]:
```suggestion
| Traffic-Global             | Net-Juniper-Mseries-Traffic-Global-custom                  | Check the bandwidth of the interfaces. For each checks the name of the interface will appear    | X          |
```

File: pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md [hunk 8]
```diff
@@ -2,48 +2,819 @@
...
+| FILTER         | Filter by peer identifier (can be regexp)                                                                                                                                                                                                                                                                                |                                                                 |             |
+| FILTERREMOTEIP | Filter by remote ip address (can be regexp)                                                                                                                                                                                                                                                                              |                                                                 |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:329]:
```suggestion
| FILTERREMOTEIP | Filter by remote IP address (can be regexp)                                                                                                                                                                                                                                                                              |                                                                 |             |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md (hunk 3 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-routers-juniper-mseries-snmp.md:68]:
```suggestion
| Net-Juniper-Mseries-Storage-Name | Découvre les partitions des disques et supervise leur taux d'occupation        |
```

**Pull request merged**